google/cloud/bigtable/tests/instance_admin_emulator.cc

```diff
@@ -36,6 +37,10 @@ class InstanceAdminEmulator final
       grpc::ServerContext* context,
       btadmin::CreateInstanceRequest const* request,
       google::longrunning::Operation* response) override {
+    std::string request_text;
+    google::protobuf::TextFormat::PrintToString(*request, &request_text);
+    std::cout << __func__ << "request=" << request_text << std::endl;
```

Comment:
This and other changes introduce a lot of debug output... was this a temporary change while debugging? Should they be removed?

Reply:
OK, SGTM.